Whiskey/Functions/New-WhiskeySemanticVersion.ps1

```diff
@@ -39,6 +44,21 @@ function New-WhiskeySemanticVersion
     {
         $semVersion = $Version | ConvertTo-WhiskeySemanticVersion -ErrorAction Stop
     }
+    elseif( $Path )
+    {
+        if($Path -like '*/package.json')
+        {
+            $semVersion = Get-Content -Raw -Path $Path | 
+                ConvertFrom-Json | 
+                Select-Object -ExpandProperty 'version' -ErrorAction Ignore | 
+                ConvertTo-WhiskeySemanticVersion -ErrorAction Stop
+        }
+        if($Path -like '*.psd1')
+        {
+            $semVersion = (Test-ModuleManifest $Path).Version | 

```

Comment:
```
$semVersion = Test-ModuleManifestPath |
    Select-Object -ExpandObject 'Version' |
    ConvertTo-WhiskeySemanticVersion -ErrorAction Stop
```

Add validation that you got a version back. If you didn't, the module manifest is either 1) bad or 2) doesn't contain a version. Write an error, e.g. `Write-Error -Message ('Unable to determine module version from module manifest ''{0}''. Please make sure this is a valid PowerShell module manifest, that it contains a Version property, and that the Version is a valid semantic version, e.g. it follow the pattern MAJOR.MINOR.PATCH.' -f $fileInfo.FullName)`.

Remove the `-ErrorAction Stop` on the `ConvertTo-WhiskeySemanticVersion` so that we can write a more helpful error message.

Reply:
Done